code/modules/awaymissions/mission_code/dsbssigma.dm

```diff
@@ -0,0 +1,702 @@
+					/obj/item/reagent_containers/glass/bottle/atropine = 1, /obj/item/reagent_containers/glass/bottle/oculine = 1, /obj/item/reagent_containers/glass/bottle/toxin = 1, /obj/item/reagent_containers/syringe/antiviral = 2,
+					/obj/item/reagent_containers/syringe/insulin = 1, /obj/item/reagent_containers/hypospray/autoinjector = 2, /obj/item/reagent_containers/food/pill/salbutamol = 2, /obj/item/reagent_containers/food/pill/mannitol = 2, /obj/item/reagent_containers/food/pill/mutadone = 1,
+					/obj/item/stack/medical/bruise_pack/advanced = 1, /obj/item/stack/medical/ointment/advanced = 1, /obj/item/stack/medical/bruise_pack = 2, /obj/item/stack/medical/splint = 2, /obj/item/reagent_containers/glass/beaker = 2, /obj/item/reagent_containers/dropper = 1,
+					/obj/item/healthanalyzer = 1, /obj/item/healthupgrade = 1, /obj/item/reagent_containers/iv_bag/slime = 1)
+	contraband = list(/obj/item/reagent_containers/glass/bottle/sulfonal = 1, /obj/item/reagent_containers/glass/bottle/pancuronium = 1)
+
+/obj/machinery/autolathe/hacked
+	hacked = TRUE
+
+/obj/machinery/autolathe/hacked/Initialize(mapload)
+    . = ..()
+    adjust_hacked(TRUE)
+
+/obj/machinery/door_control/away/button_single_use
+	var/beenused = FALSE
+	var/activation_msg = "<span class='notice'>The button briefly activates, then sparks and breaks.</span>"
+
+/obj/machinery/door_control/away/button_single_use/attack_hand(mob/user as mob)
+	if(allowed(user) && (wires & 1) && !beenused)
+		..()
+		beenused = TRUE
+		stat = BROKEN
+		playsound(get_turf(src), "sparks", 100, TRUE)
+		do_sparks(3, 1, src)
+		to_chat(user, activation_msg)
+
+/obj/machinery/door_control/away/button_single_use/dsbssigma_lockdown_random
+	name = "Emergency Lockdown Control"
+	req_access = list(ACCESS_AWAY01)
+	activation_msg = "<span class='notice'>The button briefly activates, then sparks and breaks. It seems only some sections of the base have been unlocked.</span>"
+
+/obj/machinery/door_control/away/button_single_use/dsbssigma_lockdown_random/Initialize()
+	..()
+	id = pick("dsbss_lockdown_blast_medical", "dsbss_lockdown_blast_botany", "dsbss_lockdown_blast_research")
+
+//////// MECHA ////////
+
+/obj/mecha/combat/gygax/dark/rusted
+	desc = "This once lightweight exosuit, painted in a dark scheme, has not been spared by the elements. Exterior armor is heavily rusted and most equipment sockets have decayed beyond repair."
+	name = "rusted Dark Gygax"
+	max_integrity = 200
+	deflect_chance = 10
+	armor = list(melee = 20, bullet = 20, laser = 25, energy = 20, bomb = 10, bio = 0, rad = 10, fire = 50, acid = 50)
+	max_temperature = 25000
+	leg_overload_coeff = 50
+	operation_req_access = list(ACCESS_AWAY01)
+	internal_damage_threshold = 25
+	max_equip = 0
+	destruction_sleep_duration = 2
+
+/obj/mecha/combat/gygax/dark/rusted/add_cell()
+	cell = new /obj/item/stock_parts/cell/high(src)
+
+/obj/mecha/combat/gygax/dark/rusted/onTransitZ()
+	do_sparks(3, 1, src)
```

Comment:
Best mention in the description that you cant take it out the away mission such as "Any heavy bluespace translocation is likely to heavily break it", except written in a much better way